### ISSUE TITLE
Testing skyfield 1.25 with current assay fails on s390x

### ISSUE THREAD

**Issue by bnavigator** [brandon-rhodes/assay#7]
Obscure IBM archictecture:

[s390x_log.txt](https://github.com/brandon-rhodes/assay/files/4975502/s390x_log.txt)
```
[   22s] + /usr/bin/python3 -m assay --batch skyfield.tests
[   26s] .......................................................................................................................................................................................................................................................................................................................................................................................................................................................................Traceback (most recent call last):
[   26s]   File "/usr/lib64/python3.8/runpy.py", line 194, in _run_module_as_main
[   26s]     return _run_code(code, main_globals, None,
[   26s]   File "/usr/lib64/python3.8/runpy.py", line 87, in _run_code
[   26s]     exec(code, run_globals)
[   26s]   File "/home/abuild/rpmbuild/BUILD/assay-79f5d784a55d70f31b2c90b636910738cd74840e/assay/__main__.py", line 2, in <module>
[   26s]     main()
[   26s]   File "/home/abuild/rpmbuild/BUILD/assay-79f5d784a55d70f31b2c90b636910738cd74840e/assay/command.py", line 22, in main
[   26s]     monitor.main_loop(args.name, args.batch or not isatty)
[   26s]   File "/home/abuild/rpmbuild/BUILD/assay-79f5d784a55d70f31b2c90b636910738cd74840e/assay/monitor.py", line 68, in main_loop
[   26s]     runner.send(source)
[   26s]   File "/home/abuild/rpmbuild/BUILD/assay-79f5d784a55d70f31b2c90b636910738cd74840e/assay/monitor.py", line 144, in runner_coroutine
[   26s]     give_work_to(worker)
[   26s]   File "/home/abuild/rpmbuild/BUILD/assay-79f5d784a55d70f31b2c90b636910738cd74840e/assay/monitor.py", line 129, in give_work_to
[   26s]     paths = [path for name, path in worker.call(list_module_paths)]
[   26s]   File "/home/abuild/rpmbuild/BUILD/assay-79f5d784a55d70f31b2c90b636910738cd74840e/assay/worker.py", line 77, in call
[   26s]     return pickle.load(self.from_worker)
[   26s] _pickle.UnpicklingError: pickle data was truncated
[   26s] error: Bad exit status from /var/tmp/rpm-tmp.yBjsJt (%check)
```

**Comment by bnavigator** [comment]
Replacing `assay.unix.cpu_count` with `os.cpu_count` (https://github.com/brandon-rhodes/assay/issues/8#issuecomment-663854671) fixes this, too.
https://build.opensuse.org/package/show/home:bnavigator:skyfield-debug/python-skyfield

**Comment by brandon-rhodes** [comment]
Alas, that was only added in Python 3.4. Is there any chance that f443e61ce1adedec4d73ee8e5d8bb6034e0c0c09 will fix this? I don't quite see how this error message arose from the same problem, but if `os.cpu_count()` fixed it, then f443e61ce1adedec4d73ee8e5d8bb6034e0c0c09 might fix it too!

**Comment by bnavigator** [comment]
The pickling error seems to happen occasionally on all platforms. So s390 is not different to #8
